wof/wofpy_config.py

```diff
@@ -58,6 +59,9 @@ def copytree(src, dst, symlinks=False, ignore=None, overwrite='soft'):
             continue
         s = os.path.join(src, item)
         d = os.path.join(dst, item)
+        if overwrite == 'soft' and not os.path.exists(d):
+            print('Adding {}'.format(d))
```

Comment:
This will be more verbose b/c it will always spell out loud all the files it is adding or removing.